src/main/java/org/reaktivity/nukleus/sse/internal/SseConfiguration.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright 2016-2017 The Reaktivity Project
+ *
+ * The Reaktivity Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.reaktivity.nukleus.sse.internal;
+
+import org.reaktivity.nukleus.Configuration;
+
+public class SseConfiguration extends Configuration
+{
+    public static final String INITIAL_COMMENT_ENABLED = "nukleus.sse.initial.comment.enabled";
```

Comment:
Consider allowing the size of the initial comment to be specified (rendered as spaces), rather than just on or off.

Reply:
Used the above config flag as is, but a direct buffer is precomputed for the comment.